src/main/resources/templates/find-mentor.html

```diff
@@ -33,5 +34,5 @@
 
 {% block scripts %}
 {{ super() }}
-<script charset="utf-8" type="text/javascript" src="static/scripts/find-mentor.js"></script>
+<script charset="utf-8" type="text/javascript" src="static/scripts/find-mentor.js" onload="updateButtonContainer()"></script>
```

Comment:
you can also just call the method in the js file. similar to line 69: refillMentors();